CMakeLists.txt

```diff
@@ -5,6 +5,14 @@ project (CHAKRACORE)
 # option. Otherwise, CMake fails to update cached
 # references
 
+if(CCACHE_PROGRAM_NAME)
+  find_program(CCACHE_PROGRAM ${CCACHE_PROGRAM_NAME})
```

Comment:
Super minor nit: lots of the variables in this file get passed in with `_SH` at the end to designate that they come from build.sh, so it might be good to have the variables be `CCACHE_PROGRAM` and `CCACHE_PROGRAM_SH`.

Reply:
Ok, I'll add `_SH`.